File: Angular/Hahn_Softwareentwicklung/src/app/models/paymentMethod.model.ts
export class PaymentMethodModel{
    id: string;
    name: string;

    constructor(id: string, name: string){
        this.id = id;
        this.name = name;
    }


}

export enum PaymentMethods
        {
            None,
            CreditCard,
            DebitCard,
            NetBanking,
            UPI,
            Wallet
        }

File: Backend/Models/PaymentMethodModel.cs
namespace Hahn_Softwareentwicklung.Models
{
    public class PaymentMethodModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public PaymentMethodModel(int id,string name)
        {
            Id = id;
            Name = name;
        }

        public enum PaymentMethods
        {
            None,
            CreditCard,
            DebitCard,
            NetBanking,
            UPI,
            Wallet
        }
    }
}
